### ISSUE TITLE
Last update removes the slider for 3rd monitor

### ISSUE THREAD

**Issue by HalinReno** [emoacht/Monitorian#532]
Up until the last update, my third monitor no longer has a slider for brightness. The monitor is an Acer S271HL. HDR is not supported and is not enabled in Windows 11. I have also tried disabling my other monitors but no matter what combination I have enabled, the slider does not show for the Acer monitor. I'm not exactly sure what info you need, so just let me know. Thank you!

**Comment by emoacht** [comment]
Which version you are talking about?
The recent updates don't include a modification which affects detection of monitors.
To begin with, could you read [Contributing Guidelines](https://github.com/emoacht/Monitorian/blob/9aeb4238af22c60ccc03051f144edd1a5ee9acdb/CONTRIBUTING.md) and post __probe.log__? How to produce that log is explained in [Reporting](https://github.com/emoacht/Monitorian#reporting) in readme.

**Comment by HalinReno** [comment]
[probe.log](https://github.com/emoacht/Monitorian/files/13192105/probe.log)

Thanks for your help.

**Comment by emoacht** [comment]
Thank you for the log.
ACER S271HL fails to set brightness.
Could you post __operation .log__ as well? Please wait around one hour for accumulating logs.

**Comment by HalinReno** [comment]
I started the operation log. After I did that, I decided to disable that monitor and re-enable it. When it came up, I clicked monitorian in the taskbar and the slider came up after about 2 seconds. I restarted the computer and now the slider is gone. I disabled and re-enabled the monitor but I can't get it to come back up again.
[operation.log](https://github.com/emoacht/Monitorian/files/13192416/operation.log)

The operation log is still going in case I didn't leave it going long enough.
